lib/Archive/Tar/Stream.pm

```diff
@@ -83,6 +84,7 @@ Args:
    inpos     - initial offset in infh
    outpos    - initial offset in outfh
    safe_copy - boolean.
+   no_die    - boolean.
```

Comment:
This should have some more doc in the SYNOPSIS section too explaining what it does.